artifactory/services/upload.go

```diff
@@ -321,6 +321,11 @@ func scanFilesByPattern(uploadParams UploadParams, rootPath string, progressMgr
 	// 'uploadedDirs' is in use only when we need to upload folders with flat=true.
 	// 'uploadedDirs' will contain only local directories paths that have been uploaded to Artifactory.
 	var uploadedDirs []string
+	if shouldUploadAnEmptyArchive(uploadParams.Archive, paths) {
```

Comment:
Let's move`uploadedDirs` to line 320 and add a log message that indicates JFROG_CLI_UPLOAD_EMPTY_ARCHIVE mode is detected